src/lib/GameList/GameListCard.svelte

```diff
@@ -28,33 +29,37 @@
 	const minGroup = targetGroupArr[0];
 	const maxGroup = targetGroupArr.slice(-1)[0];
 
-	const targetGroupString = `Groep ${minGroup} t/m ${maxGroup}`;
+	let targetGroupString;
+	if (minGroup === 1 && maxGroup === 8) {
+		targetGroupString = 'Alle groepen';
+	} else {
+		targetGroupString = `Groep ${minGroup} t/m ${maxGroup}`;
+	}
 </script>
 
 <article>
 	<h1>{game.name}</h1>
+	{#if game.offline}
+		<h2>Offline beschikbaar</h2>
+	{/if}
 	<ul>
-		<li>{targetGroupString}</li>
-		<li class="category-label">{game.category.name || game.category}</li>
-		<li>Min. spelers: {game.minimumPlayers}</li>
-		{#if game.offline}
-			<li class="offline">Offline Beschikbaar</li>
-		{/if}
+		<CardLabel>{targetGroupString}</CardLabel>
+		<CardLabel>{game.category.name || game.category}</CardLabel>
+		<CardLabel>
+			<i class="icon-persons" />
+			{game.minimumPlayers}
+		</CardLabel>
 	</ul>
 </article>
 
 <style>
 	article {
-		background-color: var(--color-lighter-grey);
-		border-radius: 1em;
+		background-color: var(--color-white);
+		border-radius: 1.5em;
 		margin: 1em 0;
 		padding: 1em;
 		justify-content: space-between;
-	}
-
-	li.offline {
-		background-color: var(--color-grey);
-		color: var(--color-white);
+		filter: drop-shadow(0 0.2em 0.2em hsl(120, 46%, 89%));
```

Comment:
Gaf een net wat leuker effect dan een `box-shadow`

Reply:
Misschien handig dit in een variable zetten